source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -60,18 +62,20 @@ export abstract class Dialog {
             dialogFrameElement.style.justifyContent = "space-between";
             dialogFrameElement.setAttribute("role", "dialog");
             dialogFrameElement.setAttribute("aria-modal", "true");
-            dialogFrameElement.setAttribute("aria-labelledby", "acd-dialog-title-element");
+            dialogFrameElement.setAttribute(
+                "aria-labelledby",
+                "acd-dialog-title-element"
+            );
             dialogFrameElement.tabIndex = -1;
 
             dialogFrameElement.onclick = (e) => {
                 // disable click bubbling from the frame element -- otherwise it'll get to the overlay, closing the
                 // dialog unexpectedly
-                if(this.preventDefaultAction) {
-                e.cancelBubble = true;
-
-                return false;
+                if (this.preventLightDismissal) {
```

Comment:
Why does this condition need to be here? This should have nothing to do with light dismissal?

Reply:
@dclaux  Used this condition here to identify the image dialog flow without which other input actions such as File browser, checkbox are not reaching its respective listener.